integration/app/app.component.spec.ts

```diff
@@ -379,23 +385,34 @@ describe('AppComponent', () => {
       return arr;
     };
 
-    for (let i = 0; i < 13; i++) {
+    for (let i = 0; i < 23; i++) {
```

Comment:
what the magic number 23?

Reply:
To have two full pages with 10 elements and a last page with just 3.

Reply:
bring to variable countPages

Reply:
```typescript
const entityCount = 23;
for (let i = 0; i < entityCount; i++) {
  component.addToDo();
}
// ...
component.getPaginatedEntities(10, 2).pipe(map(todos => todos.map(t => t.title)))
  .subscribe(last => {
    expect(last.length).toBe(3);
    expect(last).toEqual(generateTitles(21, entityCount));
});
```

?

Reply:
```ts
for (let i = 0, entityCount = 23; i < entityCount; i++) {
```